src/test/java/BaiduTest.java

```diff
@@ -0,0 +1,44 @@
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+import de.retest.recheck.Recheck;
+import de.retest.recheck.RecheckImpl;
+import de.retest.web.RecheckWebOptions;
+
+public class BaiduTest {
+
+	private WebDriver driver;
+	private Recheck re;
+
+	@Before
+	public void setUp() {
+		final RecheckWebOptions options = (RecheckWebOptions) RecheckWebOptions.builder() //
+				.addIgnore( "baidu.filter" ) //
```

Comment:
I understand the filters should be working on their own, thus maybe we even want to use the filters exclusive (`setIgnore`)?